cloudy_mozdef/dmake

```diff
@@ -29,13 +29,25 @@ function check_img() {
 
 check_img || exit 127
 
+if [ -n "$AWS_CONFIG_FILE" ]; then
+  config_file_mount="-v ${AWS_CONFIG_FILE}:${AWS_CONFIG_FILE}"
+fi
+
+if [ -n "$AWS_SHARED_CREDENTIALS_FILE" ]; then
+  config_file_mount="${config_file_mount} -v ${AWS_SHARED_CREDENTIALS_FILE}:${AWS_SHARED_CREDENTIALS_FILE}"
+fi
+
+dmake_env_file="`mktemp`"
+trap "{ rm -f \"$dmake_env_file\"; }" EXIT
+env | egrep "^AWS" > "$dmake_env_file"
+
+cat $dmake_env_file
```

Comment:
```suggestion
echo "Will use the following environment variables:"
cat $dmake_env_file
```

Reply:
Doh! Sorry, that was what I was using to debug and I left it in. I'll just remove it. Thanks for catching that.